schemas/input/csl-data.json

```diff
@@ -4,425 +4,477 @@
+          "description": "Used to define metadata for an original publication.",
+          "$ref": "#/definitions/bibitem"
+        },
+        "original-publisher": {
+          "description": "[Deprecated - Use `publisher` on the `original` related item instead]",
+          "type": "string"
+        },
+        "original-publisher-place": {
+          "description": "[Deprecated - Use `publisher-place` on the `original` related item instead]",
+          "type": "string"
+        },
+        "original-title": {
+          "description": "[Deprecated - Use `title` on the `original` related item instead]",
+          "type": "string"
+        },
+        "page": {
+          "type": ["string", "number"]
+        },
+        "page-first": {
+          "type": ["string", "number"]
+        },
+        "part": {
+          "type": ["string", "number"]
+        },
+        "part-title": {
+          "type": "string"
+        },
+        "PMCID": {
+          "type": "string"
+        },
+        "PMID": {
+          "type": "string"
+        },
+        "printing": {
+          "type": ["string", "number"]
+        },
+        "publisher": {
+          "type": "string"
+        },
+        "publisher-place": {
+          "type": "string"
+        },
+        "references": {
+          "type": "string"
+        },
+        "reviewed": {
+          "title": "Reviewed - Related Item",
+          "description": "In a review, defines metadata for the reviewed item.",
+          "$ref": "#/definitions/bibitem"
+        },
+        "reviewed-title": {
+          "description": "[Deprecated - Use `title` on the `reviewed` related item instead]",
+          "type": "string"
+        },
+        "scale": {
+          "type": "string"
+        },
+        "section": {
+          "type": "string"
+        },
+        "source": {
+          "type": "string"
+        },
+        "status": {
+          "type": "string"
+        },
+        "supplement": {
+          "type": ["string", "number"]
+        },
+        "title": {
+          "type": "string"
+        },
+        "title-short": {
+          "type": "string"
+        },
+        "translated-title": {
+          "description": "[Deprecated - Q: what TODO?]",
+          "type": "string"
+        },
+        "translated-title-short": {
+          "description": "[Deprecated - Q: what TODO?]",
+          "type": "string"
+        },
+        "URL": {
+          "type": "string"
+        },
+        "version": {
+          "type": "string"
+        },
+        "volume": {
+          "type": ["string", "number"]
+        },
+        "year-suffix": {
+          "type": "string"
+        },
+        "custom": {
+          "title": "Custom key-value pairs.",
+          "type": "object",
+          "description": "Used to store additional information that does not have a designated CSL JSON field. The note field can also store additional information, but custom is preferred for storing key-value pairs.",
```

Comment:
```suggestion
          "description": "Used to store additional information that does not have a designated CSL JSON field. The custom field is preferred over the note field for storing custom data, particularly for storing key-value pairs, as the note field is used for user annotations in annotated bibliography styles.",
```

Reply:
nice change. we should consider implementing it separately from this PR, since it's unrelated.